lib/buildsystems/meson.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Meson < Package
+
+  def self.meson_options(options = '')
+    return (@meson_options = options if options)
+  end
+
+  def self.check?(bool = true)
+    return (@check = bool)
+  end
+
+  def self.build
+    system "meson #{CREW_MESON_OPTIONS} #{@meson_options} builddir"
```

Comment:
How would you override `CREW_MESON_OPTIONS`?

Reply:
My initial thought is that if you want to deviate from the `buildsystems` setup, then just don't use them, and then the package is just like in status quo ante...

Changes which are used often enough can be migrated as an option into `buildsystems`...

Reply:
Made an issue for dealing with this later: https://github.com/chromebrew/chromebrew/issues/8434 , since this isn't necessary to get the larger functionality in place.